python/langwatch/src/langwatch/litellm.py

```
@@ -0,0 +1,359 @@
+            self.client._original_completion = self.client.completion  # type: ignore
+            self.client._original_acompletion = self.client.acompletion  # type: ignore
+
+            self.client.completion = self.patched_completion_create  # type: ignore
+            self.client.acompletion = self.patched_completion_acreate  # type: ignore
+
+    def patched_completion_create(self, *args, **kwargs):
+        trace = None
+        try:
+            trace = get_current_trace()
+        except:
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)